### PR TITLE
Autocomplete in the table browser in SQL lab is broken

### PR DESCRIPTION
**Pull request by khtruong**
### CATEGORY

Choose one

- [x] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
The autocomplete in the table browser in SQL lab does not work. We use the createFilterOptions API from the react-select-fast-filter-options library for autocomplete. The API expects a list of option objects where its properties **value** and **label** are strings. The bug is that we were passing objects to the property **value**, which caused the algorithm to not work properly. Now we explicitly pass strings to the API.

### TEST PLAN
Manually verified

### REVIEWERS
@betodealmeida @DiggidyDave @xtinec

### CODE REVIEW

File: superset/assets/src/components/TableSelector.jsx [hunk 1]
```diff
@@ -130,7 +130,12 @@ export default class TableSelector extends React.PureComponent {
           `${encodeURIComponent(schema)}/${encodeURIComponent(substr)}/${forceRefresh}/`);
        return SupersetClient.get({ endpoint })
         .then(({ json }) => {
-          const filterOptions = createFilterOptions({ options: json.options });
+          const filterOptions = createFilterOptions({
+              options: json.options.map(o => ({
```
Comment (betodealmeida) [inline superset/assets/src/components/TableSelector.jsx]: @khtruong, I think the indentation here is wrong and will probably trip the linter.

Comment (khtruong) [reply superset/assets/src/components/TableSelector.jsx]: Hmm. I ran the linter and it was green. Do you prefer it formatted differently?

Comment (etr2460) [reply superset/assets/src/components/TableSelector.jsx]: I don't think the linter actually checks for this (maybe we should add it) but in general the JS code uses 2 space indents

Comment (khtruong) [reply superset/assets/src/components/TableSelector.jsx]: Ohhh. Brain fart. I wasn't seeing what you meant. Yes, let me update this to 2 spaces.

**Comment by betodealmeida** [comment]
Awesome!

**Pull request merged**